pkg/util/http.go

```diff
@@ -109,8 +109,11 @@ func (l Log) logWithRequest(r *http.Request) log.Logger {
 			localLog = log.With(localLog, "sourceIPs", ips)
 		}
 	}
-
-	return user.LogWith(r.Context(), localLog)
+	orgID := r.Header.Get(user.OrgIDHeaderName)
```

Comment:
Can you fallback on getting in the context then on `anonymous` ? There's some cases where the header won't be there specially on the query path.

Reply:
I've added fallback. Don't know how to test it though.